contrib/launch-venv.sh

```diff
@@ -31,7 +31,8 @@ for var in $(env | grep FWUPD | cut -d= -f1); do
         ENV="${ENV} ${var}=${!var}"
 done
 SUDO=$(which sudo)
-if [ "${BIN}" = "fwupd" ] && \
+if [ "${BIN}" = "fwupd-binder" ] || \
+   [ "${BIN}" = "fwupd" ] && \
```

Comment:
can you please rebase this, we had some changes in main.

Reply:
Done